platforms/hdp2.6.5/docs/HDP-installation.md

```diff
@@ -181,20 +150,21 @@ Information" are in the configurations.
 <img src="docs/review.jpeg" alt="dependent_config.jpeg" width="400" height="200">
 
 7. Create HDFS folders:
-hadoop fs -mkdir /user/splice
-hadoop fs -mkdir /user/splice/history
-hadoop fs -mkdir /user/splice/spark-warehouse
 
-hadoop fs -chmod 1777 /user/splice
-hadoop fs -chmod 1777 /user/splice/history
-hadoop fs -chmod 755 /user/splice/spark-warehouse
+`hadoop fs -mkdir /user/splice`
+`hadoop fs -mkdir /user/splice/history`
+`hadoop fs -mkdir /user/splice/spark-warehouse`
+
+`hadoop fs -chmod 1777 /user/splice`
+`hadoop fs -chmod 1777 /user/splice/history`
+`hadoop fs -chmod 755 /user/splice/spark-warehouse`
 
-hadoop fs -chown hbase:hbase /user/splice
-hadoop fs -chown hbase:spark /user/splice/history
-hadoop fs -chown hbase:hbase /user/splice/spark-warehouse
+`hadoop fs -chown hbase:hbase /user/splice`
+`hadoop fs -chown hbase:spark /user/splice/history`
```

Comment:
Why does `/user/splice/history` have different group assigned?

Reply:
I don't know the source of that difference but we tested the procedure this way.